tests/Java.Interop-PerformanceTests/Java.Interop-PerformanceTests.csproj

```diff
@@ -11,9 +11,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="nunit" Version="3.11.0" />
```

Comment:
Is this a sign that we should move all of these `@(PackageReference)`s into a new `build-tools/scripts/nunit.targets` file, so that we don't need to update *12* `.csproj` files the next time we want to bump NUnit versions?

Reply:
I guess it comes down to what workflow we want to optimize for: IDE tooling or hand editing?

The current way is easy for IDE tooling, you click the package and hit Upgrade All and it's all done for you.  Moving it to a targets file would mean you could no longer update from an IDE.

On the other hand, a targets file is definitely easier for editing by hand rather than updating 12 files manually.

Unfortunately there does not seem to be a good solution that optimizes for both workflows.